package.json

```diff
@@ -848,9 +848,9 @@
     "vscode-nls": "^2.0.1"
   },
   "devDependencies": {
-    "@types/node": "^10.17.24",
+    "@types/node": "^13.11.0",
     "@types/tmp": "0.0.33",
-    "@types/vscode": "^1.30.0",
+    "@types/vscode": "^1.46.0",
```

Comment:
The version of @types/vscode is [supposed to match the version](https://code.visualstudio.com/api/working-with-extensions/testing-extension#migrating-from-vscode) in the `engine` part of package.json, which basically specifies the minimum VSCode version for this extension. We want to keep that version as low as possible so that even folks with old installs can use it. We can raise the minimum VSCode version if we want to implement a feature that's only available in a new version, but I don't think that's the case so far, though I'm not testing it on old versions and I doubt anyone is running a version older than 6 months.

Anyway, I say we leave this at 1.30.0 unless there's a reason to bump it up.

Reply:
Actually, those are devDependencies. They are used for compiling the TypeScript code into JavaScript, so they are seen only for the developers. Since VScode only sees the final JavaScript, there is no impact on the engine itself.
We catch more bugs on newer versions.

Reply:
Actually, you are right 😃 
I'll revert the change to this file